packages/release-mastermind/types/index.d.ts

```diff
@@ -3,0 +1,266 @@
+import {
+  Condition,
+  IssueCondition,
+  PRCondition,
+  ProjectCondition,
+  ScheduleCondition
+} from '../src/conditions'
+import { Repo } from '../src/utils'
+
+/**
+ * Application interfaces
+ */
+
+export interface Options {
+  configPath: string
+  configJSON: Runners
+  showLogs: boolean
+  dryRun: boolean
+  fillEmpty: boolean
+  skipDelete: boolean
+  repo?: Repo
+}
+
+export interface Runners {
+  labels?: Labels
+  runners: Config[]
+}
+
+export interface Config {
+  root: string
+  versioning: {
+    source: VersionSource
+    type?: VersionType
+  }
+  retryLimit?: number
+  prereleaseName?: string
+  labels?: { [key: string]: string }
+  sharedConfig?: SharedConfig
+  pr?: PullRequestConfig
+  issue?: IssueConfig
+  project?: ProjectConfig
+  schedule?: ScheduleConfig
+}
+
+/**
+ * Config types
+ */
+
+export type VersionSource = 'node' | 'milestones' | string
+export type VersionType = 'SemVer'
+
+export interface SharedConditions {
+  requires: number
+  conditions: Condition[]
+}
+
+export interface Label {
+  name: string
+  description: string
+  color: string
+}
+
+export interface Labels {
+  [key: string]: Label
+}
+
+export interface PRConditionConfig {
+  requires: number
+  conditions: PRCondition[]
+}
+
+export interface IssueConditionConfig {
+  requires: number
+  conditions: IssueCondition[]
+}
+
+export interface ProjectConditionConfig {
+  requires: number
+  conditions: ProjectCondition[]
+}
+
+export interface ScheduleConditionConfig {
+  requires: number
+  conditions: ScheduleCondition[]
+}
+
+export interface PullRequestConfig extends SharedConfig {
+  assignProject?: AssignProject[]
+  automaticApprove?: AutomaticApprove
+  manageRelease?: Release
+  duplicateHotfix?: { [title: string]: DuplicateHotfix }
+  syncRemote?: SyncRemote[]
+}
+
+export interface IssueConfig extends SharedConfig {
+  assignProject?: AssignProject[]
+  createBranch?: { [label: string]: CreateBranch }
```

Comment:
Codacy has a fix for the issue: [Expected a semicolon.](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
  createBranch?: { [label: string]: CreateBranch };
```